common/djangoapps/util/tests/test_db.py

```diff
@@ -192,6 +193,7 @@ def test_used_ids(self, times):
             self.assertIn(int_id, list(set(range(minimum, maximum + 1)) - used_ids))
 
 
+@pytest.mark.skip("NUTMEG UPGRADE: restore this when we de-fork edx-organizations for good")
```

Comment:
A string comment can be difficult to find. Please make a specific setting for skipping all Tahoe nutmeg tests.

This variable should be `False` in `common.py` and `True` only in testing environments. This variable is similar to `TAHOE_ALWAYS_SKIP_TEST` but is meant to be removed in the short term -- unlike `TAHOE_ALWAYS_SKIP_TEST`.

```suggestion
@pytest.mark.skipIf(
     settings.TAHOE_NUTMEG_TEMP_SKIP_TEST, 
     "restore this when we de-fork edx-organizations for good"
)
```

Reply:
Done